cmd/nginx-ingress/main.go

```diff
@@ -262,21 +261,23 @@ func main() {
 			port := fmt.Sprintf(":%v", *readyStatusPort)
 			s := http.NewServeMux()
 			s.HandleFunc("/nginx-ready", ready(lbc))
-			glog.Fatal(http.ListenAndServe(port, s))
+			l.Log(ctx, levels.LevelFatal, fmt.Sprint(http.ListenAndServe(port, s))) // nolint:gosec
+			os.Exit(0)
```

Comment:
```suggestion
			os.Exit(1)
```
Exit 0 is a success exit code